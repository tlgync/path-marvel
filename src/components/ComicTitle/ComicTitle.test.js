import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { ComicTitle } from '.';

test('ComicTitle Render SnapShot', () => {
  const title = 'Avengers: The Initiative (2007) #19';
  const tree = renderer.create((
    <Router>
      <ComicTitle title={title} />
    </Router>
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
