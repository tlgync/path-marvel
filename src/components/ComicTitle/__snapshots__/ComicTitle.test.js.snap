// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComicTitle Render SnapShot 1`] = `
<div
  className="comic-title"
>
  <span>
    Avengers: The Initiative (2007) #19
  </span>
</div>
`;
